bccsp/pkcs11/pkcs11.go

```diff
@@ -51,46 +52,37 @@ func loadLib(lib, pin, label string) (*pkcs11.Ctx, uint, *pkcs11.SessionHandle,
 		}
 	}
 	if !found {
-		return nil, slot, nil, fmt.Errorf("Could not find token with label %s", label)
+		return nil, slot, nil, fmt.Errorf("could not find token with label %s", label)
 	}
 
-	session := createSession(ctx, slot, pin)
-
-	logger.Debugf("Created new pkcs11 session %+v on slot %d\n", session, slot)
-
-	if pin == "" {
-		return nil, slot, nil, fmt.Errorf("No PIN set")
-	}
-	err = ctx.Login(session, pkcs11.CKU_USER, pin)
```

Comment:
Why is this logic removed? It's not mentioned in the PR description... if it must be removed I'd suggest remove in a separate PR or add the motivation to the PR description.

Reply:
It's already performed in `createSession` it's accidentally duplicated code